Makefile

```diff
@@ -48,6 +52,8 @@ GOARCH = $(word 2, $(platform_temp))
 # Set default base image dynamically for each arch
 ifeq ($(GOARCH),amd64)
 		DOCKERFILE ?= Dockerfile-$(BIN)
+local-arch:
```

Comment:
`local-arch` - For Power, this target would run a script that would download the Restic binary

As we are creating arch images (except amd64) without using dedicated arch nodes this will be needed (also in case of downloading the binary from Restic GH releases in future), this can be used in the future while downloading Restic binary for arm platforms.

Reply:
I see that L66-68 are doing the restic download, however still unclear on the purpose of L55-56

Reply:
L55-56 is added to avoid the workflow failure due to local-arch target missing while building for amd64 as in this case, it will be downloaded via Dockerfile. L155 will always trigger local-arch target and L55-56 will help us avoid any failure.

Do let me know if any further clarification is needed on this.